### ISSUE TITLE
How to start the Postgis DB?

### ISSUE THREAD

**Issue by pol-todo** [urbangrammarai/data_processing#36]
Hi @darribas ,

Question about running the code: What port, username, and password do we put to connect to the Postgis db in the `gds_dev` flavour of gds? I couldn't find this on the urban-grammar documentation, or the gds docs (i see the  base gds_py flavour has postgres listed in the conda deps: https://darribas.org/gds_env/stacks/gds_py/). 

I even checked the env variables within the docker image, and no values are associated with `DB_USER` etc.
- https://urbangrammarai.xyz/data_processing/code/Load_data_from_PostGIS.html

The documentation says run:

```bash
docker run \
    -it \
    -e DB_PORT=<port> \
    -e DB_USER=<username> \
    -e DB_PWD=<password> \
    -e DB_HOST=<host> \
    darribas/gds_dev:7.0
```

Then we can run this to set the env vars for the jupyter notebooks.

```bash
  import os
  
  import geopandas as gpd
  
  from sqlalchemy import create_engine
  
  user = os.environ.get('DB_USER')
  pwd = os.environ.get('DB_PWD')
  host = os.environ.get('DB_HOST')
  port = os.environ.get('DB_PORT')
```
Which print out empty values.

Perhaps I am overlooking something? 

Any help would be most appreciated, would love to use this in our lab's research. Looks awesome!

Pol

**Comment by darribas** [comment]
One for @martinfleis , but from memory we had a PostGIS instance running _separately_ from the GDS environment. That had a table for the features in OS Local. Martin might have a pointer to documentation on how to set up that instance.

**Comment by pol-todo** [comment]
@darribas Record response time! 

Ahhhh, ok. That will make things easier. It sounds like even this will work? 
- https://github.com/postgis/docker-postgis

**Comment by darribas** [comment]
Yes, i think we did use a containerised version. This is how we inserted the table:

https://urbangrammarai.xyz/data_processing/code/Fetch_OS_Data.html#save-geodataframe-to-postgis

**Comment by martinfleis** [comment]
This was GDLS-wide PostGIS instance someone else was setting up, so I don't have the details how. But if I was doing this today, I'd use the container, if I needed the database. 

Given it is used within this project just for storage, I would just use static files instead. The DB was used to have these data shared across the lab.

**Comment by pol-todo** [comment]
@martinfleis Thanks so much for the update. I'll give these a try!
